lib/chef/provider/user.rb

```diff
@@ -153,11 +156,7 @@ def compare_user
           new_val = new_resource.send(user_attrib)
           cur_val = current_resource.send(user_attrib)
           if !new_val.nil? && new_val.to_s != cur_val.to_s
-            if user_attrib.to_s == "password" && new_resource.sensitive
-              @change_desc << "change #{user_attrib} from ******** to ********"
-            else
-              @change_desc << "change #{user_attrib} from #{cur_val} to #{new_val}"
-            end
```

Comment:
This block needs to be kept as it is. It is to mask password property when it is set as sensitive.

Reply:
Hi, yeah it was some regression on my side which I asked John about. Fixed in the following commit - thanks for spotting!